### PR TITLE
LRQA-67491 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-67491
I ran them here [melodywu#33](https://github.com/melodywu/liferay-portal/pull/33).

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/PortalSettings.macro [hunk 1]
```diff
@@ -914,48 +914,35 @@ definition {
 		AssertElementPresent(locator1 = "PortalSettingsAuthentication#AUTHENTICATION_OPEN_SSO_TEST_OPEN_SSO_CONFIGURATION_BUTTON");
 	}
 
-	macro viewConfigurationUsersDefaultUserAssociationsOrganizationSitesCP {
+	macro viewConfigurationUsersDefaultUserAssociationsCP {
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/PortalSettings.macro:917]: @melodywu Would we still need this macro if you updated the tests to just use `PortalSettings.editConfigurationUsersDefaultUserAssociationsCP`?

Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/macros/PortalSettings.macro:917]: I think we don't need it.

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
https://issues.liferay.com/browse/LPS-144665

**Comment by patriciajeanperez** [comment]
ci:test:sf

**Comment by melodywu** [comment]
> ci:test:sf

Thanks

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**